.github/pull_request_template.md

```diff
@@ -38,5 +38,8 @@ If any of these are not completed, please explain why in the notes.
 - [ ] My changes don't break anything unexpected
 - [ ] I have checked and updated the security.txt file where required
 - [ ] Up to date with the main branch
-- [ ] I have a commit with the fix:, feat or BREAKING_CHANGE: tags and will copy this commit message to the squash
+- [ ] I have a commit with the fix:, feat or BREAKING_CHANGE: tags (or other relevant tag) and will copy this commit message to the squash
 commit message
+
+Note when merging, squash commit must begin with one of the following tags:
```

Comment:
Could you add examples of the format and use cases?  When would we use "chore" (if we don't expect to use it don't mention in the list).  

E.g 
**fix:** - use when you are fixing an issue that has been found in the existing solution
example commit message - _fix: Missing link on Help Centre page_